viewing_party/main.py

```diff
@@ -0,0 +1,249 @@
+def create_movie(title, genre, rating):
+    """
+    Creates a dictionary representing the description of a movie
+    """
+    movie_dict = {
+        "title": title,
+        "genre": genre,
+        "rating": rating
+    }
+
+    if movie_dict["title"] is None or movie_dict["genre"] is None or\
+            movie_dict["rating"] is None:
+        return None
+    return movie_dict
+
+
+def add_to_watched(user_data, movie):
+    """
+    Creates a list of watched movies.
+    """
+    user_data["watched"] = []
```

Comment:
Because user_data["watched"] is already provided as an empty list I don't think you need to 'remake' it as an empty list again.